src/app/public/modules/restricted-view/restricted-view.directive.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  Directive,
+  TemplateRef,
+  ViewContainerRef,
+  Renderer2,
+  OnDestroy
+} from '@angular/core';
+
+import {
+  SkyRestrictedViewAuthService
+} from './auth.service';
+
+import {
+  Subject
+} from 'rxjs';
+
+import 'rxjs/add/operator/takeUntil';
```

Comment:
Are you able to have this library use the newer rxjs versions where this is piped in instead of affecting the prototype?